openshift/buildConfig-template.yaml

```diff
@@ -28,38 +28,28 @@ objects:
         name: "kebechet-job:${IMAGE_STREAM_TAG}"
     runPolicy: Serial
     source:
-      git:
-        uri: ${GITHUB_URL}
-        ref: ${GITHUB_REF}
       type: Git
+      git:
+        uri: "https://github.com/harshad16/kebechet.git"
+        ref: "feature/dockerfile"
     strategy:
-      type: Source
-      sourceStrategy:
-        from:
-          kind: ImageStreamTag
-          name: python-36-centos7:latest
+      type: Docker
+      dockerStrategy:
+        dockerfilePath: Dockerfile
         env:
-        - name: ENABLE_PIPENV
-          value: '1'
-        - name: UPGRADE_PIP_TO_LATEST
-          value: ''
+        - name: USER
+          value: 'kebechet'
+        - name: PIPENV_CACHE_DIR
+          value: '/home/user'
+        - name: LC_ALL
+          value: 'en_US.UTF-8'
+        - name: LANG
+          value: 'en_US.UTF-8'
```

Comment:
``USER``, ``LC_ALL`` and ``LANG`` are redundant here. You already have them in Dockerfile.